packages/react/src/components/Icon/Icon.base.tsx

```diff
@@ -64,16 +64,34 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
     );
     const containerProps = hasName
       ? {
-          role: isImage ? undefined : 'img',
-          'aria-label': isImage ? undefined : accessibleName,
+          role: isImage || mergeImageProps ? undefined : 'img',
+          'aria-label': isImage || mergeImageProps ? undefined : accessibleName,
         }
       : {
           'aria-hidden': true,
         };
 
+    let finalIconContentChildren = iconContentChildren;
+
+    if (mergeImageProps && finalIconContentChildren && typeof finalIconContentChildren === 'object') {
+      finalIconContentChildren = React.cloneElement(iconContentChildren, {
+        alt: this.props.title,
```

Comment:
Would it make sense to do `alt: accessibleName` here, since it'll then respect `ariaLabel`/`aria-label` if passed, and will still fall back on title?

Reply:
The problem is that a value passed for `aria-label` is not semantically appropriate for `alt`.